### PR TITLE
tidied front_content.php

### PR DESCRIPTION
**Pull request by gonatan**
Please have a close look at my changes as I didn't have a running system atm.

I also added two TODO token w/ questions about the current code. Maybe you have a clue ..

### CODE REVIEW

File: contenido/includes/frontend/include.front_content.php [hunk 1]
```diff
@@ -62,12 +62,12 @@
 $backendPath = cRegistry::getBackendPath();
 $backendUrl  = cRegistry::getBackendUrl();
 
+// Include cronjob-Emulator
 if ($cfg['use_pseudocron'] == true) {
-    // Include cronjob-Emulator
-    $oldpwd = getcwd();
```
Comment (gonatan) [inline contenido/includes/frontend/include.front_content.php:67]: I checked for the usage of "$oldpwd" in other files too.

File: contenido/includes/frontend/include.front_content.php [hunk 2]
```diff
@@ -314,14 +311,14 @@
...
-} else {
+if (isset($idart)) {
     $oArtLang = new cApiArticleLanguage($idartlang);
     $online   = $oArtLang->get('online');
+} else {
+    $online = ('0' !== $oCatLang->get('visible'));
 }
 
-$idartlang = getArtLang($idart, $lang);
+$idartlang = getArtLang($idart, $lang); // TODO do we need to call getArtLang() here again?
```
Comment (gonatan) [inline contenido/includes/frontend/include.front_content.php]: Lines 313 and 321 are identical. Probably due to a merge.

File: contenido/includes/frontend/include.front_content.php [hunk 3]
```diff
@@ -694,18 +688,28 @@
...
             // default redirection is temporary
             // with status code 302 or 307 (since HTTP/1.1)
-            $protocol      = $_SERVER['SERVER_PROTOCOL'];
-            $redirect_code = ($protocol === 'HTTP/1.1') ? 307 : 302;
             if ($redirect_mode === 'permanently') {
-                $redirect_code = 301;
+                if ($_SERVER['SERVER_PROTOCOL'] === 'HTTP/1.1') {
+                    $redirect_code = 301; // Moved Permanently
+                    // TODO shouldn't it be 308?
+                    // $redirect_code = 308; // Permanent Redirect
```
Comment (gonatan) [inline contenido/includes/frontend/include.front_content.php]: Not sure why we didn't make this distinction for permanent redirects.

File: contenido/includes/frontend/include.front_content.php (hunk 2 above)
Comment (muratpurc) [reply contenido/includes/frontend/include.front_content.php]: Thanks for the hint.

File: contenido/includes/frontend/include.front_content.php (hunk 3 above)
Comment (muratpurc) [reply contenido/includes/frontend/include.front_content.php]: Permanent redirect should use 308.

**Review by muratpurc — APPROVED** [review]

**Pull request merged**